app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -3187,14 +3246,19 @@ export class Dispatcher {
     )
   }
 
-  private logHowToRevertSquash(tip: IValidBranch): void {
+  private logHowToRevertMultiCommitOperation(
```

Comment:
I tried to reuse this one for rebase but I saw some differences, so I stopped there since I didn't want to break anything (else) 😂